### DISCUSSION TITLE
Add docking capabilities to Modal

### DISCUSSION THREAD

**Discussion by LoupaLoupa** [Blazored/Modal#502] [Q&A]
I was wondering if anyone has tried to add docking feature to modal so you can have it free floating and offer possibilities to dock it in a component if user wants to.

I have been trying to see how I can do that, I think it could be an interesting feature, but I wonder where to start.

Any help would appreciated.

Many thanks!

**Comment by chrissainty** [comment]
I'd be curious what scenario would require docking to another component? And what that means?

**Reply by LoupaLoupa** [reply]
Well, I have been using Blazored Modal to have floating simulated electrical instruments on my Wasm app. This works very well!  I was asked if we could give the opportunity to the users to keep them floating or dock and undock the tools on the surface of the main page, so I guess I did not mean another components, but more like the current page showed in the SPA.

Not sure if that makes any sense, but it does in my head 😁
